pl_bolts/datamodules/binary_mnist_datamodule.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         drop_last: bool = False,
         *args: Any,
         **kwargs: Any,
-    ) -> None:
```

Comment:
@akihironitta, isn't it best practice to keep return type as `None`, if the function is not returning anything?

Reply:
I vaguely remember reading that for methods with typed arguments and no return, the best practice is to not have return type `None`. I might be completely off base, so please let me know if I should add `None` for return types.

Reply:
@briankosw 
> I vaguely remember reading that for methods with typed arguments and no return, the best practice is to not have return type `None`.

I see. It would be nice to have any resources regarding this as I'm not aware very much.

As I read [PEP 484 - Type Hints](https://www.python.org/dev/peps/pep-0484/) though, annotating `None` seems reasonable.
> the return type of `__init__` ought to be annotated with `-> None`.

Reply:
I can't seem to find the resource, and I think it's best practice to prefer explicit over implicit code, so I'll annotate them.